projects/optic/src/init.ts

```diff
@@ -54,6 +54,7 @@ export const initCli = async (
   } = {}
 ): Promise<Command> => {
   cli.name('optic');
+  cli.helpOption('-h, --help', 'Display help for the command');
```

Comment:
overrides the default option thats appended to everything with the same text, but capitalized